app/controllers/api/articles_controller.rb

```diff
@@ -43,13 +43,13 @@ def update
   private
 
   def attach_image(article)
-    params[:article][:image].present? && DecodeService.attach_image(params[:article][:image].first, article.image)
+    params[:article][:image].present? && DecodeService.attach_image(params[:article][:image], article.image)
```

Comment:
So it's working now without the array? :D

Reply:
yes, as soon as deployed we will see images on frontend, checked on localhost.